docs/validators/setup/statesync.md

```diff
@@ -0,0 +1,188 @@
+<!--
+order: 4
+-->
+
+# State Sync
```

Comment:
nice, let's also link the [Snapshots & Archive Nodes](https://docs.evmos.org/validators/snapshots_archives.html) page so that users can sync their nodes